lib/jsdom/utils.js

```diff
@@ -277,3 +277,19 @@ exports.visitTree = function (root, callback) {
   }
   visit(root);
 };
+
+function isValidAbsoluteURL(str) {
```

Comment:
Put a `// TODO URL: be more rigorous` here so @Sebmaster can remember to base this on whatwg-url when that gets merged.